force_wfmanager/ui/setup/new_entity_creator.py

```diff
@@ -70,6 +70,13 @@ class NewEntityCreator(HasStrictTraits):
     #: A message to be displayed if there are no config options
     _no_config_options_msg = ReadOnly(Unicode)
 
+    #: Option for the user to manually control the visibility of the
+    #: configuration option display
+    config_visible = Bool(True)
+
```

Comment:
Including new options for controlling the configuration view (used by `MCOParameterView`) and displaying a header to make clear what model options are being displayed